f8a_report/manifest_main.py

```diff
@@ -0,0 +1,28 @@
+"""Dynamic Manifest Generation."""
+
+import logging
+from datetime import datetime as dt, timedelta
+from report_helper import ReportHelper
+from manifest_helper import manifest_interface
+import os
+
+logger = logging.getLogger(__file__)
+
+
+def main():
+    """Generate the dynamic manifest file weekly."""
+    r = ReportHelper()
+    today = dt.today()
+
+    start_date_wk = (today - timedelta(days=7)).strftime('%Y-%m-%d')
+    end_date_wk = today.strftime('%Y-%m-%d')
+
+    logger.info(os.environ.get('GENERATE_MANIFESTS', 'False'))
+    if os.environ.get('GENERATE_MANIFESTS', 'False') in ('True', 'true', '1'):
```

Comment:
Do we need the condition now ?

Reply:
Can we control this via Flag in may be some other way..?

Reply:
This job will be scheduled to run once a week. The flag will be able to decide whether the we need to generate the manifest or not.

Reply:
We can use cron schedule value to control whether to run the job or not. Isn't the additional flag is redudant?

Reply:
It is more convenient to control via a flag than by using the cron schedule. I have seen multiple issues with this generation so it is useful